### DISCUSSION TITLE
Example problem with Newton-Raphson

### DISCUSSION THREAD

**Discussion by avtarsinghh1991** [idaholab/moose#20458] [Q&A Modules: General] [answered]
Hello MOOSE Experts

I am working with some non-linear equations which I need to solve using the Newton-Raphson procedure. I just want to know if there is any example equation (should have kernels) that is solved using the Newton Raphson method. Thank you very much.

Best
Avtar

**Comment by GiudGiud** [comment, marked as answer]
Hello

Sorry we missed this. 
Just set solve_type = 'NEWTON' in your executioner. 
see https://mooseframework.inl.gov/source/executioners/Steady.html

Guillaume
